frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.tsx

```diff
@@ -212,14 +211,18 @@ const usePreviewPublisher = (): PreviewPublisherContextType => {
       return;
     }
 
-    publisherRef.current = initPublisher(undefined, { insertDefaultUI: false }, (err: unknown) => {
-      if (err instanceof Error) {
-        publisherRef.current = null;
-        if (err.name === 'OT_USER_MEDIA_ACCESS_DENIED') {
-          console.error('initPublisher error: ', err);
+    publisherRef.current = initPublisher(
+      undefined,
+      { insertDefaultUI: false, resolution: '1280x720' },
```

Comment:
Can we double check this is doing what we expect on real devices too?

Reply:
I updated the test instructions to test on real devices. FWIW, it looked okay on my iOS device